.gitattributes

```diff
@@ -68,55 +39,29 @@ service/pipelines/api.go linguist-generated=true
 service/pkg.go linguist-generated=true
 service/provisioning/api.go linguist-generated=true
-service/provisioning/credentials_usage_test.go linguist-generated=true
-service/provisioning/encryption_keys_usage_test.go linguist-generated=true
 service/provisioning/impl.go linguist-generated=true
 service/provisioning/interface.go linguist-generated=true
 service/provisioning/model.go linguist-generated=true
-service/provisioning/networks_usage_test.go linguist-generated=true
-service/provisioning/private_access_usage_test.go linguist-generated=true
-service/provisioning/storage_usage_test.go linguist-generated=true
-service/provisioning/vpc_endpoints_usage_test.go linguist-generated=true
-service/provisioning/workspaces_usage_test.go linguist-generated=true
 service/serving/api.go linguist-generated=true
 service/serving/impl.go linguist-generated=true
 service/serving/interface.go linguist-generated=true
 service/serving/model.go linguist-generated=true
 service/settings/api.go linguist-generated=true
 service/settings/impl.go linguist-generated=true
 service/settings/interface.go linguist-generated=true
-service/settings/ip_access_lists_usage_test.go linguist-generated=true
 service/settings/model.go linguist-generated=true
-service/settings/token_management_usage_test.go linguist-generated=true
-service/settings/tokens_usage_test.go linguist-generated=true
-service/settings/workspace_conf_usage_test.go linguist-generated=true
 service/sharing/api.go linguist-generated=true
 service/sharing/impl.go linguist-generated=true
 service/sharing/interface.go linguist-generated=true
 service/sharing/model.go linguist-generated=true
-service/sharing/providers_usage_test.go linguist-generated=true
-service/sharing/recipients_usage_test.go linguist-generated=true
-service/sharing/shares_usage_test.go linguist-generated=true
-service/sql/alerts_usage_test.go linguist-generated=true
 service/sql/api.go linguist-generated=true
-service/sql/dashboards_usage_test.go linguist-generated=true
-service/sql/data_sources_usage_test.go linguist-generated=true
 service/sql/impl.go linguist-generated=true
 service/sql/interface.go linguist-generated=true
 service/sql/model.go linguist-generated=true
-service/sql/queries_usage_test.go linguist-generated=true
-service/sql/query_history_usage_test.go linguist-generated=true
-service/sql/statement_execution_usage_test.go linguist-generated=true
-service/sql/warehouses_usage_test.go linguist-generated=true
 service/workspace/api.go linguist-generated=true
-service/workspace/git_credentials_usage_test.go linguist-generated=true
 service/workspace/impl.go linguist-generated=true
 service/workspace/interface.go linguist-generated=true
 service/workspace/model.go linguist-generated=true
-service/workspace/repos_usage_test.go linguist-generated=true
-service/workspace/secrets_usage_test.go linguist-generated=true
-service/workspace/workspace_usage_test.go linguist-generated=true
```

Comment:
Do you know why the test ignores are gone?

If not, the change can be reverted to not mess with it, and we can deal with it elsewhere.

Reply:
@pietern it was because SDK was regenerated without fix in openapi/roll code from this PR, fixed now